cmd/webhook/admission/testdata/ingress/valid-ingress-with-annotations.json

```diff
@@ -0,0 +1,41 @@
+{
+    "request": {
+        "uid": "req-uid",
+        "name": "req1",
+        "namespace": "n1",
+        "object": {
+            "metadata": {
+                "name": "ing1",
+                "namespace": "ing1",
+                "annotations": {
+                    "zalando.org/skipper-filter": "status(10) -> inlineContent(\"This should work\")",
+                    "zalando.org/skipper-predicate": "Header(\"test\", \"test\") && Path(\"/login\")",
+                    "zalando.org/skipper-routes": "r1: Header(\"test2\", \"test2\") && Path(\"/login\") -> status(200) -> \"shunt\""
```

Comment:
Why is `"shunt"` not wrong?

Reply:
Why should it? It's a valid eskip, right now that's what we do

Reply:
Lets put a valid value here e.g. `http://foo.test` to reduce confusion, see also https://www.rfc-editor.org/rfc/rfc6761

> Why is "shunt" not wrong?

We'd need to tighten backend validation, lets tackle it in a separate PR.

Reply:
Yes we can do this but we should catch also these kind of missing URL, in case we assume `network` backend we should have a `scheme` and `host` (or IP). "shunt" might match as a network backend today, but this is not "valid" in a sense from user pov, so we should not have this in tests, that we want not to test "wrong backend".